Profile-Card-2/opensource.css

```diff
@@ -1,128 +1,131 @@
-
```

Comment:
Don't make changes in other contributions if there is no issue created about it.

Reply:
Ok